### PR TITLE
Feature/tvist1 437 hearing deadlines etc

### PR DESCRIPTION
**Pull request by jekuaitk**
https://jira.itkdev.dk/browse/TVIST1-437

* Adds all hearing response deadline logic
* Adds more restrictions to when a hearing response may be created

### CODE REVIEW

**Review by rimi-itk — CHANGES_REQUESTED** [review]
Looks good. Some minor changes needed.

File: src/Entity/CaseEntity.php [hunk 1]
```diff
@@ -732,4 +743,28 @@ public function setValidatedAt(?\DateTimeInterface $validatedAt): self
...
+    }
+
+    public function setHearingResponseDeadline(?\DateTimeInterface $hearingResponseDeadline): self
+    {
+        $this->hearingResponseDeadline = $hearingResponseDeadline;
+
+        return $this;
+    }
+
+    public function getHasReachedHearingResponseDeadline(): ?bool
```
[the remark this answers is not in the document]
Comment (jekuaitk) [reply src/Entity/CaseEntity.php]: Updated all boolean getters in 7326761d4ef857a3c6c7e7c40f79535e4cc31b5c.

[the remark this answers is not in the document]
Comment (jekuaitk) [reply src/Entity/CaseEntity.php]: Woops, i was a little to quick here. This created a problem in our [logging](https://github.com/itk-dev/naevnssekretariatet/blob/feature/TVIST1-437-hearing-deadlines-etc/src/Logging/EntityListener/AbstractEntityListener.php#L134-L146).

[the remark this answers is not in the document]
Comment (jekuaitk) [reply src/Entity/CaseEntity.php:759]: Reverted 7326761d4ef857a3c6c7e7c40f79535e4cc31b5c.

**Review by rimi-itk — APPROVED** [review]

File: src/EventSubscriber/PostPersist.php [hunk 2]
```diff
@@ -1,18 +1,21 @@
...
+use App\Service\DocumentUploader;
+use App\Service\MailTemplateHelper;
 use Doctrine\Bundle\DoctrineBundle\EventSubscriber\EventSubscriberInterface;
 use Doctrine\ORM\EntityManagerInterface;
 use Doctrine\ORM\Event\LifecycleEventArgs;
 use Doctrine\ORM\Events;
 use Symfony\Contracts\Translation\TranslatorInterface;
 
-class ReceiptHelper implements EventSubscriberInterface
+class PostPersist implements EventSubscriberInterface
```
[the remark this answers is not in the document]
Comment (jekuaitk) [reply src/EventSubscriber/PostPersist.php]: Renamed to `EntityPostPersistSubscriber` in 52570e2078cbb7a507edfc25b60a62f9fe0edf61.

**Pull request merged**